includes/class-activitypub.php

```diff
@@ -247,6 +252,7 @@ public static function template_redirect() {
 	 */
 	public static function add_query_vars( $vars ) {
 		$vars[] = 'activitypub';
+		$vars[] = 'preview';
```

Comment:
I don't know if we should use a core variable here, it already gets used for post previews in a `?p=1&preview=true` format. Are we leveraging something out of that? Or could we use our own query var here like `activitypub_preview`?

Reply:
Sounds good!